WWTExplorer3d/ReferenceFrame.cs

```diff
@@ -362,6 +369,7 @@ public virtual void SaveToXml(XmlTextWriter xmlWriter)
                 xmlWriter.WriteAttributeString("Lat", Lat.ToString());
                 xmlWriter.WriteAttributeString("Lng", Lng.ToString());
                 xmlWriter.WriteAttributeString("Altitude", Altitude.ToString());
+                xmlWriter.WriteAttributeString("AltUnits", AltUnits.ToString());
```

Comment:
Same questions as in #185 here. If an older version of the software opens a file that we create, will we get the right results? I think to make sure that happens, the Altitude always needs to be serialized in meters. Then if the units label is present, we de-scale to make everything self-consistent again.

Reply:
That's a good point - if we want an older version of the software to give correct results, we'll need to use meters since it won't look at the AltUnits field.